hoomd/md/methods.py

```diff
@@ -877,154 +875,58 @@ class brownian(_Method):
 
     You can specify :math:`\gamma` in two ways:
 
-    1. Use :py:class:`set_gamma()` to specify it directly, with independent values for each particle type in the system.
-    2. Specify :math:`\lambda` which scales the particle diameter to :math:`\gamma = \lambda d_i`. The units of
-       :math:`\lambda` are mass / distance / time.
-
-    :py:class:`brownian` must be used with integrate.mode_standard.
+    1. Use ``set_gamma`` to specify it directly, with independent values for each particle type in the system.
+    2. Specify :math:`\alpha` which scales the particle diameter to :math:`\gamma = \alpha d_i`. The units of
+       :math:`\alpha` are mass / distance / time.
 
     *kT* can be a variant type, allowing for temperature ramps in simulation runs.
 
     A :py:class:`hoomd.compute.thermo` is automatically created and associated with *group*.
 
     Examples::
 
-        all = group.all()
-        integrator = integrate.brownian(group=all, kT=1.0, seed=5)
-        integrator = integrate.brownian(group=all, kT=1.0, dscale=1.5)
-        typeA = group.type('A')
-        integrator = integrate.brownian(group=typeA, kT=hoomd.variant.linear_interp([(0, 4.0), (1e6, 1.0)]), seed=10)
+        all=hoomd.filter.All()
+        brownian = hoomd.md.methods.Brownian(filter=all, kT=0.2, seed=1, alpha=1.0)
+        integrator = hoomd.md.Integrator(dt=0.001, methods=[brownian], forces=[lj])
 
     """
-    def __init__(self, group, kT, seed, dscale=False, noiseless_t=False, noiseless_r=False):
+    def __init__(self, filter, kT, seed, alpha=None):
 
-        # initialize base class
-        _Method.__init__(self)
+        # store metadata
+        param_dict = ParameterDict(
+            filter=_ParticleFilter,
+            kT=Variant,
+            seed=int(seed),
+            alpha=OnlyType(float, allow_none=True),
+            )
+        param_dict.update(dict(kT=kT, alpha=alpha, filter=filter))
+        #set defaults
+        self._param_dict.update(param_dict)
 
-        # setup the variant inputs
-        kT = hoomd.variant._setup_variant_input(kT)
+        gamma = TypeParameter('gamma', type_kind='particle_types',
+                              param_dict=TypeParameterDict(1., len_keys=1)
+                              )
 
-        # create the compute thermo
-        hoomd.compute._get_unique_thermo(group=group)
+        gamma_r = TypeParameter('gamma_r', type_kind='particle_types',
+                                param_dict=TypeParameterDict((1., 1., 1.), len_keys=1)
+                                )
+        self._extend_typeparam([gamma,gamma_r])
```

Comment:
We should just call `super().__init__(filter, kT, seed, alpha)`, and remove the `tally_reservior_energy` from the `self._param_dict` afterwards. It will be shorter and reduce code duplication.